frontend/actions/build.gradle

```diff
@@ -11,6 +11,11 @@ dependencies {
   implementation project(':frontend:base')
 }
 
+checkerFramework {
+  checkers += 'org.checkerframework.checker.tainting.TaintingChecker'
```

Comment:
CMIIW, but since we're basically using `TaintingChecker` wherever `I18NFormatterChecker` is used, I'd just rename `applyI18nFormatterChecker` to `applyI18nFormatterAndTaintingCheckers` (and include `TaintingChecker` in that function, ofc)